editoast/src/views/timetable/similar_trains.rs

```diff
@@ -1585,4 +1617,160 @@ mod tests {
         };
         assert_eq!(response, expected_response);
     }
+
+    #[test]
+    fn test_rolling_stock_characteristics_deserialize_with_name_only() {
+        let json = r#"{"name": "rolling_stock_name"}"#;
+        let result = serde_json::from_str::<RollingStockCharacteristics>(json);
+        assert!(result.is_ok());
+        let characteristics = result.unwrap();
+        assert_eq!(characteristics.name, Some("rolling_stock_name".to_string()));
+        assert_eq!(characteristics.speed_limit_tag, None);
```

Comment:
```suggestion
        let result = serde_json::from_str::<>(json);
        assert_eq!(result, Ok(RollingStockCharacteristics { name: Some("rolling_stock_name".to_string()), speed_limit_tag: None }));
```

can be made much shorter

Reply:
The `serde_json::error::Error` type does implement `PartialEq` and `Eq` as shown in the [serde_json documentation](https://docs.rs/serde_json/latest/src/serde_json/error.rs.html#17), so I can't directly use the result in comparisons. However, I improved the test by using `RollingStockCharacteristics`. Thank you for the suggestions!

Reply:
Arf... 🙄 Unwrap it is then. A few notes: 

* you don't need to `assert!(result.is_ok())` if you `unwrap` just after
* only this test has been changed

Reply:
Done.